karma-shared.conf.js

```diff
@@ -17,7 +17,7 @@ module.exports = function(config, specificOptions) {
       testName: specificOptions.testName || 'AngularJS',
       startConnect: true,
       options: {
-        // We need selenium version +2.46 for Firefox 39 and the last selenium version for OS X is 2.45.
+        // We need selenium version +2.46 for 'Firefox' 39 and the last selenium version for OS X is 2.45.
```

Comment:
I don't think so. The change is accidental. Sublime CTRL+D and ' in the wrong file